src/numJewelsInStones.js

```diff
@@ -29,6 +29,16 @@
  */
 function numJewelsInStones(jewels, stones) {
   // write code here
+  let count = 0;
+  if (jewels) {
+    const clearJewels = jewels.match(/[A-Za-z]/g).join('');
+    for (let i = 0; i < stones.length; i++) {
+      if (clearJewels.includes(stones[i])) {
+        count++;
+      }
+    }
+    return count;
+  }
+  return 0;
```

Comment:
нет надобности в двух return, если условие не пройдет, то count так и останется равен нулю 
```suggestion
  return count;
```